src/main/webapp/matchdashboard.html

```diff
@@ -13,9 +13,6 @@
     <title>Document</title>
   </head>
   <body>
-    <!-- ========================================================
-        Menu
-         ======================================================== -->
```

Comment:
FYI, there are multiple elements here with the `id` of `"user-profile"`. It's invalid HTML. I know you didn't touch that but it'd be good to fix. I tried leaving a comment the PR that introduced that too.

Reply:
I can't fix that, since I would need to modify the css too and since obi did it in scss he has told me that he wanted to do all the scss changes.